client/src/components/Ratings/index.jsx

```diff
@@ -27,23 +20,29 @@ function getPercentageRecommended(results) {
   const trues = copy.filter(
     (resultItem) => (resultItem === true)
   );
-  return (trues.length / results.length) * 100;
+  return Math.round((trues.length / results.length) * 100);
 }
 
 function averageToNearestTenth(ratings) {
-
   const total = ratings.reduce(
     (sum, item) => sum + item.rating,
     0,
   );
-  return (total / ratings.length).toFixed(1);
+
+  if (total) {
+    return (total / ratings.length).toFixed(1);
+  }
+
+  return "";
 }
 
+
 // WIP to filter by rating
 function filterByRating(stars) {
   console.log(`You clicked to filter ${stars} stars!`);
 }
```

Comment:
remove WIP functions or make them return something
also remove console logs in your final commit